Okta/OktaApi.swift

```diff
@@ -11,20 +11,25 @@
  */
 
 open class OktaApi: NSObject {
-    class func post(_ url: URL, headers: [String: String]?, postData: String?, callback: @escaping ([String: Any]?, OktaError?) -> Void) {
+    class func post(_ url: URL, headers: [String: String]?,
+        postData: String?,
+        callback: @escaping ([String: Any]?, OktaError?) -> Void) {
+
         // Generic POST API wrapper
-        
+
         var request = URLRequest(url: url)
         request.httpMethod = "POST"
         request.allHTTPHeaderFields = headers
-        
+        request.addValue(
+            "okta-sdk-appauth-ios/\(VERSION) iOS/]\(UIDevice.current.systemVersion)",
+            forHTTPHeaderField: "X-Okta-Agent"
```

Comment:
Any reason not to use the regular user agent header?

Reply:
Good catch. We *should* be using **X-Okta-User-Agent-Extended**. This is used in the [Sign-In Widget](https://github.com/okta/okta-signin-widget/blob/9c4b79d17b2763c596b6eacd628f396b13598905/src/widget/OktaSignIn.js#L174) and [Auth.js](https://github.com/okta/okta-auth-js/blob/master/lib/http.js#L29). I'll update the Android AppAuth library to reflect this change.